xhype/xhype/src/vmexit.rs

```diff
@@ -13,6 +16,104 @@ pub enum HandleResult {
     Next,
 }
 
+// Table 27-3
+#[inline]
+pub fn vmx_guest_reg(num: u64) -> X86Reg {
+    [
```

Comment:
This seems like a better fit for a match:

    match num {
        0 => X86Reg::RAX,
        (and so forth),
        _ => panic!("bad register in exit qualification"),
    }

Reply:
I updated this part with `match`.